include/ignition/physics/ConstructEmpty.hh

```diff
@@ -64,13 +74,18 @@ class ConstructEmptyModelFeature : public virtual Feature
 };
 
 /////////////////////////////////////////////////
+/// \brief This feature constructs an empty Link entity and return its pointer
+/// from the index-specified Model.
```

Comment:
I'm not sure what you mean by "index-specified". I would say "from the given Model."